src/app/modules/core/manager/landlord/components/list/landlord-list.component.ts

```diff
@@ -31,14 +32,14 @@ export class LandlordListComponent implements OnInit {
   expandedElement: Elements | null
   landlords$: Observable<Landlord[]>
 
-  constructor(private landlordService: LandlordService) {}
+  constructor(private landlordService: LandlordService, private route: ActivatedRoute) {}
 
   ngOnInit(): void {
     this.getAll()
   }
 
   getAll(): void {
-    this.landlords$ = this.landlordService.getAll()
+    this.landlords$ = this.route.snapshot.data.list
```

Comment:
`this.route.snapshot.data.list` non ritorna un Observable infatti nella view hai rimosso la pipe `async`.